src/core/entity.js

```diff
@@ -2,6 +2,15 @@ import { create_float_buffer, create_index_buffer } from './context';
 import { vec3, mat4, quat } from './math';
 import { hex_to_vec } from '../utils';
 
+const DIR_KEYS = {
```

Comment:
I would like to keep this on the Entity class. Right now in one of my prototypes, when I want the entity to move forward, backward, and sideways, but not up or down, I simply do
```js
const e = new Entity();
delete e.dir_desc[81];
delete e.dir_desc[69];
```
Also - this is the easiest way to change controls. This isn't the smartest way, but it's the simplest for now.